NodeOpera/Dockerfile.txt

```diff
@@ -0,0 +1,42 @@
+USER root
+
+#=========
+# Opera
+#=========
+ENV OPERA_VERSION=60.0.3255.170
+ENV OPERA_DRIVER_VERSION=v.2.45
+
+RUN wget -O- http://deb.opera.com/archive.key | apt-key add - && \
+     echo 'deb http://deb.opera.com/opera-stable/ stable non-free' > /etc/apt/sources.list.d/opera-blink.list
+
+RUN apt-get update
+
+RUN apt-get update && \
```

Comment:
don't need to run `apt-get update` again.

Reply:
Deleted.